fendermint/docker/docker_entry.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script should be used as ENTRYPOINT for fendermint docker image.
+
+CMD=$1
+
+if [[ $CMD == 'ipc-cli' ]]; then
+  /usr/local/bin/ipc-cli "${@:2}"
+else
+  if (( $# == 0)); then
+    /usr/local/bin/fendermint run
```

Comment:
```suggestion
    exec fendermint run
```

Reply:
I wonder if this case is even possible, with the dockerfile specifying `"run"` as the default `CMD`.

Reply:
Yes it's possible! Tested. Thanks!